.github/workflows/test.yml

```diff
@@ -44,5 +44,11 @@ jobs:
               # Share the store between the workers speeds things up further
               export OSBUILD_TEST_STORE=/var/tmp/osbuild-test-store
           fi
+
+          # Hacky replacement of container storage driver:
+          # The default overlayfs doesn't work in the runner, so let's change
+          # it to vfs for the local storage skopeo stage test.
```

Comment:
So I think the runner just doesn't support `overlay`, I think this was a workaround Achilleas added to get around this.
Rather than making the storage driver configurable

Reply:
You cannot nest `overlayfs`, right? If the test is running in a container, `/var` is probably already an `overlayfs`, so you cannot use `/var/lib/containers` as a base for yet another `overlayfs`.